### ISSUE TITLE
Month/Year/Day Labels

### ISSUE THREAD

**Issue by fablet** [sandlbn/django-bootstrap-calendar#16]
the current templates do not display the name of the month, year or day you are looking at, as appropriate to month, year or day views.
I can see this displayed in demo's for the calendar this was originally based on, but cannot find the means to put this on anywhere in the code.

**Comment by anderspetersson** [comment]
@fablet did you manage to figure out how to solve this?

**Comment by fablet** [comment]
Yes as a matter of fact I did.  There is a function in the calendar.js file called getTitle.  I created div on the calendar.html page and then set a document.getElementById equal to the getTitle and put it between the  this._loadEvents(); and this._render(); calls that instantiate the calendar.  You can tweak how the titles display between the getTitle function and the title_ strings below defaults at the top of the calendar

**Comment by daveseco7** [comment]
@fablet can you explain a little better the changes that u did , please ? 'Cause my instance is having the same problem :(

**Comment by fablet** [comment]
document.getElementById("id of the div you create on calendar.html where you want titles to show up").innerHTML = this.getTitle();

**Comment by pmaturure3** [comment]
Hi Fablet, i would like to implement the same, could u please share an explicit example on how you did this
